lib/igniter/libs/swoosh.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Igniter.Libs.Swoosh do
+  @moduledoc "Codemods & utilities for working with Swoosh"
+
+  @doc "Lists all mailers found in the project."
+  @spec list_mailers(Igniter.t()) :: {Igniter.t(), [module()]}
+  def list_mailers(igniter) do
+    Igniter.Project.Module.find_all_matching_modules(igniter, fn _mod, zipper ->
+      move_to_mailer_use(igniter, zipper) != :error
+    end)
+  end
+
+  @doc "Moves to the use statement in a module that matches `use Swoosh.Mailer`"
+  @spec move_to_mailer_use(Igniter.t(), Sourceror.Zipper.t()) ::
+          :error | {:ok, Sourceror.Zipper.t()}
+  def move_to_mailer_use(_igniter, zipper) do
```

Comment:
I have mixed feelings on requiring an unused argument here.

It's consistent in that all the functions in `Libs.Phoenix` and `Libs.Ecto` require it, *but* all those functions actually use the argument. Maybe this function starts using it in the future in order to be more intelligent or efficient, but I somewhat doubt it. 

We should probably either:

1. Remove the argument and make it `move_to_mailer_use/1`, or
2. Change the signature to `move_to_mailer_use(%Igniter{}, zipper)` to at least ensure the right thing is being passed in, even if it's not being used.

I think 1. is preferable (we can always deprecate it later), but I could be convinced of 2.

Reply:
I'm fine with 1. I just copied over some of the code from the phoenix module last week and made it work. 
I should have taken another look at it today before pushing.